src/test/java/nova/core/util/EnumExample.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package nova.core.util;
+
+/**
+ * @author ExE Boss
+ */
+public enum EnumExample {
```

Comment:
This change was done to allow testing of `EnumIdentifier` using my `EnumExample` class.